cli/tests/unit/data/job_runs.txt

```diff
@@ -0,0 +1,3 @@
+JOB ID   ID                            STARTED AT           
```

Comment:
can we center `ID` here or left align the other columns? 
Same with the other tables